include/inviwo/core/util/formats.h

```diff
@@ -30,18 +30,14 @@
 #pragma once
 
 #include <inviwo/core/common/inviwocoredefine.h>
-#include <inviwo/core/util/glmvec.h>
+#include <inviwo/core/util/glm.h>
```

Comment:
consider glmvec/glmmat/glmutils if full glm functionality is not required.

Reply:
I see

Reply:
maybe those should be in a separate header to... the rarely used...